updateVersion.sh

```diff
@@ -105,6 +105,7 @@ function merge_files() {
   AUTH_SRC=$SRC/istio-auth
   ISTIO=$ROOT/install/kubernetes/istio.yaml
   ISTIO_AUTH=$ROOT/install/kubernetes/istio-auth.yaml
+  ISTIO_AUTH_PER_CLUSTER_CA=$ROOT/install/kubernetes/templates/istio-auth/isio-auth-per-cluster-ca.yaml
```

Comment:
This file includes everything (mixer, manager etc). It thus feels wrong to put it under template/istio-auth